### PR TITLE
Fixes an issue where ads are not correctly centered on certain platforms.

### LINKED ISSUE #5958: [amp-ad] Shifted amp-ad on Android 4.2.2 system browsers
Same error has been seen on desktop Safari 7 as well. 

screenshot_20161101-154454

### PR DESCRIPTION
**Pull request by glevitzky**
Fixes #5958.

### CODE REVIEW

File: ads/google/doubleclick.js [hunk 1]
```diff
@@ -53,6 +54,15 @@ export function doubleclick(global, data) {
     };
   }
 
+  // Center the ad in the container.
+  setStyles(global.document.querySelector('#c'), {
+    top: '50%',
+    left: '50%',
+    bottom: '',
+    right: '',
+    webkitTransform: 'translate(-50%, -50%)',
```
Comment (jridgewell) [inline ads/google/doubleclick.js]: Just `transform`.

Comment (glevitzky) [inline ads/google/doubleclick.js]: I don't understand. The use of transform is the issue that needs correcting.

Comment (jridgewell) [inline ads/google/doubleclick.js]: Yes, but `setStyle` will correctly [map](https://github.com/ampproject/amphtml/blob/master/src/style.js#L96-L97) `transform` to `webkitTransform`.

Comment (glevitzky) [inline ads/google/doubleclick.js]: Ah, neat. Now I understand what you meant by getVendorJsPropertyNa,e.

File: ads/google/doubleclick.js [hunk 2]
```diff
@@ -53,6 +54,18 @@ export function doubleclick(global, data) {
     };
   }
 
+  // Center the ad in the container.
+  const container = global.document.querySelector('#c');
+  if (container) {
```
Comment (jridgewell) [inline ads/google/doubleclick.js]: When wouldn't there be a container?

Comment (jridgewell) [inline ads/google/doubleclick.js]: Let's add a `dev().assertElement` then? Can we do that from ads code?

Comment (glevitzky) [inline ads/google/doubleclick.js]: I will try.

Comment (glevitzky) [inline ads/google/doubleclick.js]: Do you mean to add `dev().assertElement(container)` in addition to checking `if(container) { ...`?

**Pull request merged**